components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -51,9 +46,19 @@
           <div class="filter-label">Unknown</div>
           <div class="filter-total">{{ serviceGroupsHealthSummary.unknown }}</div>
         </chef-option>
+        <chef-option class="filter connected" value='connected' (click)="updateHealthFilter('connected')" (keyup.enter)="updateHealthFilter('connected')">
```

Comment:
Would improve the code to tighten the signature of `updateHealthFilter` to accept only `ServiceGroupsFacadeService.allowedStatus` (plus `"total"` with a union type) instead of a general string... but that would require changing `allowedStatus` to be an enum, I suspect, so may be out of scope for this PR. 🤷

Reply:
might as well update it while i'm in here.  I've updated now and would appreciate your feedback prior to merging

Reply:
Getting there, but we really only need a single source of truth.
Should be able to eliminate `AllowedHealthStatus` completely and just use the `allowedStatus` enum directly (should capitalize to `AllowedStatus` though). Also, should then delete the entire line `health = includes(health, this.allowedHealthStatus) ? health : allowedStatus.TOTAL;` in `updateHealthFilter`; no longer need to check that constraint.

Reply:
Thanks for the thoughts - I don't think I realized enums could be used as a type signature 👍